spec.bs

```diff
@@ -2535,17 +2564,23 @@ interface InterestGroupReportingScriptRunnerGlobalScope
         : InterestGroupScriptRunnerGlobalScope {
   undefined sendReportTo(DOMString url);
   undefined registerAdBeacon(record&lt;DOMString, USVString&gt; map);
+  undefined registerAdMacro(DOMString name, USVString value);
 };
 
 </pre>
 
+Note: Currently, registerAdMacro is only available in reportWin(), but not reportResult().

```

Comment:
please link to `reportWin()` and `reportResult()` functions here. Also is this going to change soon? I notice you said "Currently"

Reply:
Removed the word "Currently". I confirmed with the team that there's no plan of supporting it in reportResult() since there's no more asks from people for that (at least at this moment).